bundles/net.tourbook/src/net/tourbook/database/TourDataUpdate.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (C) 2022 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.database;
+
+import net.tourbook.data.TourData;
+
+/**
+ * Class to update tour values for {@link TourData}
+ */
+public abstract class TourDataUpdate {
+
+   private int    databaseVersion;
+   private String splashManagerMessage;
```

Comment:
After using a common splash message, there is no need to save the splash message in `TourDataUpdate`

The message can be used directly in `TourDatabase`

![image](https://user-images.githubusercontent.com/1283445/164882774-c27b6a7f-3d77-48b8-b7b9-6d5e38b1145b.png)


The text key should be adjusted, currently it is `Tour_Database_PostUpdate_048_Weather_Clouds`

As I already suggested, we should not use a formatting mix of NLS and String formatting replacements
`Data update %d: %d of %d - %d % - %d Δ`

Reply:
Ok, I will do a text key that is generic and remove it from the tour data updater.

Reply:
Fixed